content/webapp/pages/exhibition-guide.tsx

```diff
@@ -134,16 +154,16 @@ const Stops: FC<StopsProps> = ({ stops, type }) => {
                       audioFile={audioWithoutDescription.url}
                     />
                   )}
-                {type === 'bsl' && bsl?.embed_url && (
-                  <VideoEmbed embedUrl={bsl.embed_url} />
+                {type === 'bsl' && bsl?.embedUrl && (
```

Comment:
I just changed this to `embed_url` because the types were broken. Was that a mistake?